### DISCUSSION TITLE
Empty Best Performance?

### DISCUSSION THREAD

**Discussion by memesnightmares** [ppy/osu#21830] [Q&A] [answered]
I play osu lazer (almost) exclusively, but my lazer.ppy.sh profile doesn't list any of my best performance plays (either from stable or on lazer, if you look at my recent plays you'll see an 80pp that would be in my top 100 easily if set on stable). When I look at any other profiles, they do seem to show top plays. Any idea what's going on there?

Here's my profile: https://lazer.ppy.sh/users/14613516

Update: Tried setting a play on stable (thought the issue might be related to my account being "inactive" (no rank, 0pp) but that didn't seem to fix it. Did bump me up 50k ranks (on osu.ppy.sh) for seemingly no reason.

Update: Now my profile page is fully bugged. It gives "Oh no! Something broke! ;_;" when I load it, example error code `de425ca4c4594cbcba30fa80bc627421`.

**Comment by isuewo** [comment]
I'm having the same issue, you are also on Linux, right?

**Reply by memesnightmares** [reply]
Yep, I would guess it's web side though, probably not a client issue? Since neither stable or lazer shows show up? I also do have a rank and PP, but just empty Best Performance section (and new plays don't seem to add to PP).

**Reply by isuewo** [reply]
Seems like we're having the same issue then

**Reply by memesnightmares** [reply]
Resolved for you now? Or are new scores still not showing up?

**Reply by isuewo** [reply]
Best Performances seem to also be resolved for me, rank and pp count are still empty for me though :(

**Comment by peppy** [comment, marked as answer]
This should now be resolved.

**Reply by memesnightmares** [reply]
Kind-of, see discussion on https://github.com/ppy/osu-web/issues/9686. You're welcome to close out this discussion to avoid duplication between this and 9686 if you'd like.
